src/pages/home/library/EditSong.tsx

```diff
@@ -127,46 +289,58 @@ const EditSong = () => {
-              gap: 4,
-              justifyContent: ["center", "center", "normal"],
+      <Box pt={ 5 } pb={ 7 }>
+        <WizardForm
+          initialValues={ initialValues }
+          onSubmit={ handleSubmit }
+          rootPath={ `home/library/edit-song/${songId}` }
+          isProgressStepperVisible={ true }
+          validateOnMount={ true }
+          enableReinitialize={ true }
+          routes={ [
+            {
+              element: <BasicSongDetails isInEditMode={ true } />,
+              path: "",
+              progressStepTitle: "Basic details",
+              navigateOnSubmitStep: false,
+              onSubmitStep: handleSongInfo,
+              validationSchema: Yup.object().shape({
+                coverArtUrl: validations.coverArtUrl,
+                audio: validations.audio,
+                title: validations.title,
+                genres: validations.genres,
+                moods: validations.moods,
+                owners: validations.owners,
+                description: validations.description,
+              }),
             },
-            ".Mui-selected": {
-              background: theme.gradients[colorMap[tab]],
-              backgroundClip: "text",
-              color: theme.colors[colorMap[tab]],
-              textFillColor: "transparent",
+            {
+              element: <AdvancedSongDetails />,
+              onSubmitStep: handleAdvancedDetails,
+              path: "advanced-details",
+              progressStepTitle: "Advanced details",
+              validationSchema: Yup.object({
+                isrc: validations.isrc,
+                barcodeType: validations.barcodeType,
+                barcodeNumber: validations.barcodeNumber,
+                copyright: validations.copyright,
+                publicationDate: validations.publicationDate,
+                releaseDate: validations.releaseDate,
+                userIpi: validations.userIpi,
+                iswc: validations.iswc,
+              }),
             },
-            ".MuiTabs-indicator": {
-              background: theme.gradients[colorMap[tab]],
+            {
+              element: <ConfirmAgreement />,
+              path: "confirm",
+              progressStepTitle: "Distribute & Mint",
+              validationSchema: Yup.object().shape({
+                consentsToContract: validations.consentsToContract,
+              }),
             },
-          } }
-        >
-          <Tab label="Info" id="tab-0" aria-controls="tabpanel-0" />
-          <Tab label="Minting" id="tab-1" aria-controls="tabpanel-1" />
-          <Tab label="Marketplace" id="tab-2" aria-controls="tabpanel-2" />
-        </Tabs>
-      </Stack>
-
-      <TabPanel value={ tab } index={ 0 }>
-        <SongInfo />
-      </TabPanel>
-      <TabPanel value={ tab } index={ 1 }>
-        <MintSong />
-      </TabPanel>
-      <TabPanel value={ tab } index={ 2 }>
-        Marketplace content goes here
-      </TabPanel>
```

Comment:
1. Is this a temporary removal, to be discussed with design/product as well? The current layout does seem a little strange as a one to one copy from the Upload Song Flow. 
2. I think some copy changes can help direct the user better in this edit flow, mainly for the first page `Basic Details`. Adding additional words like _Update Song File_ might be the minimum needed for MVP.
3. I think the UPLOAD A SONG heading gave the Upload Song Flow more context in that area, same as the previous tab structure of INFO/MINTING, might want to find a way to include EDIT/MINTING in the top heading for general context?